acceptance/variables_windows.go

```diff
@@ -0,0 +1,10 @@
+package acceptance
+
+const (
+	containerBaseImage = "" // Not used in Windows tests
```

Comment:
Maybe something like
```
type RuntimeEnv interface {
  BinaryExtension() string
  ContainerBaseImage() string
}

// in test
test := "fromImage=" + runtime.ContainerBaseImage()
```
The windows implementation of that interface could throw `NotImplemented`, which would be much more clear when someone runs into a test that interfaces with a helper that isn't yet supported.

Reply:
Ah, you are right! This is an outgrowth of the time we tried to make the analyzer acceptance tests work on Windows. I can clean it up...

Reply:
Removed unneeded var